lib/device_map.py

```diff
@@ -1422,6 +1422,17 @@
                     'fastboot': False,
                     'mach': 'mvebu'}
 
+chip_pro = {'device_type': 'sun5i-gr8-chip-pro',
+                    'templates': ['generic-uboot-tftp-ramdisk-template.jinja2'],
+                    'kernel_defconfig_blacklist': [],
+                    'defconfig_blacklist': ['allmodconfig',
+                                            'multi_v7_defconfig+CONFIG_LKDTM=y'],
+                    'kernel_blacklist': [],
```

Comment:
Kevins point was they you need to blacklist older kernels, as there's no point trying to boot it on anything older than 4.10.
Please have a look at all the kernels we boot here: https://kernelci.org/job/ and blacklist any that are known to not work, thanks